pxc-57/dockerdir/entrypoint.sh

```diff
@@ -392,3 +392,6 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 fi
 
 exec "$@" $wsrep_start_position_opt
+if [ -n "$ENABLE_AUTH_PAM" ]; then
+  exec "nscld"
```

Comment:
I belive such a code cannot work at all.
exec command in 394 will never run lines 395-397

I think it is needed to run nscld inside sidecar container.

Reply:
@delgod Yes, you are correct. I verified this doesn't work as intended, but it's also not required to get auth_pam working, so I have removed. The package change and README change are still valid.